src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+const ListItem = ({ name }) => {
+  return <StyledListItem role="listitem">{name}</StyledListItem>
+}
+
+const StyledListItem = styled.p`
```

Comment:
Why is this a paragraph and not an `li` if the component name is `ListItem`?

Reply:
Fixed